### PR TITLE
Add checks for undefined to graphql result

### PR DESCRIPTION
**Pull request by AdamLiechty**

### CODE REVIEW

**Review by sfdctaka — APPROVED** [review]
LGTM!

**Review by Taost — APPROVED** [review]

**Comment by dbreese** [comment]
Would be nice to see tests for these in accountRelatedContacts tests to give better branch coverage.

File: force-app/main/default/lwc/accountRelatedContacts/accountRelatedContacts.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ export default class AccountRelatedContacts extends NavigationMixin(
   })
   graphqlResult({ data /* errors */ }) {
     this.contacts = null;
-    if (data && data.uiapi.query.Account) {
+    if (data && data.uiapi && data.uiapi.query && data.uiapi.query.Account) {
```
Comment (maliroteh-sf) [inline force-app/main/default/lwc/accountRelatedContacts/accountRelatedContacts.js:57]: can't we just use `?` propagation for the check like `if (data?.uiapi?.query?.Account) { ... }`

Comment (AdamLiechty) [reply force-app/main/default/lwc/accountRelatedContacts/accountRelatedContacts.js]: Great suggestion. Adopted.d

**Comment by AdamLiechty** [comment]
> Would be nice to see tests for these in accountRelatedContacts tests to give better branch coverage.

@dbreese good call. Added.

**Pull request merged**